History.txt

```diff
@@ -971,7 +971,7 @@ Changes between V7.4.2 and V7.5.0 released July 19 2013
 	+ Microsemi SmartFusion2
 
 	New FreeRTOSConfig.h settings
-	http://shop.freertos.org/FreeRTOS_API_and_Configuration_Reference_s/1822.htm
+	https://freertos.org/Documentation/RTOS_book.html

```

Comment:
Please like to https://freertos.org/a00110.html rather than the book here.

Reply:
done